src/utils/bloodglucose.js

```diff
@@ -124,11 +127,11 @@ export function generateBgRangeLabels(bgPrefs, opts = {}) {
   }
 
   return {
-    veryLow: `below ${thresholds.veryLowThreshold} ${bgUnits}`,
-    low: `between ${thresholds.veryLowThreshold} - ${thresholds.targetLowerBound} ${bgUnits}`,
-    target: `between ${thresholds.targetLowerBound} - ${thresholds.targetUpperBound} ${bgUnits}`,
-    high: `between ${thresholds.targetUpperBound} - ${thresholds.veryHighThreshold} ${bgUnits}`,
-    veryHigh: `above ${thresholds.veryHighThreshold} ${bgUnits}`,
+    veryLow: `${t('below')} ${thresholds.veryLowThreshold} ${bgUnits}`,
+    low: `${t('between')} ${thresholds.veryLowThreshold} - ${thresholds.targetLowerBound} ${bgUnits}`,
+    target: `${t('between')} ${thresholds.targetLowerBound} - ${thresholds.targetUpperBound} ${bgUnits}`,
+    high: `${t('between')} ${thresholds.targetUpperBound} - ${thresholds.veryHighThreshold} ${bgUnits}`,
+    veryHigh: `${t('above')} ${thresholds.veryHighThreshold} ${bgUnits}`,
```

Comment:
The order may be different in another language. Instead of simply translate the text, it should be parameterized. This is true for every changes in this `return`.
Ex:
```suggestion
    veryHigh: t('above {{thresholds.veryHighThreshold}} {{bgUnits}}', {thresholds, bgUnits}),
```

Reply:
agree but we have to create new variables to get it working in all the cases.

Reply:
> agree but we have to create new variables to get it working in all the cases.

What do you mean ?

Reply:
got it done